tests/tx/test_verification.py

```diff
@@ -0,0 +1,448 @@
+        ):
+            self.manager.verification_service.verify_basic(tx)
+
+        # Transaction methods
+        verify_parents_basic_wrapped.assert_called_once()
+        verify_weight_wrapped.assert_called_once()
+        verify_pow_wrapped.assert_called_once()
+        verify_number_of_inputs_wrapped.assert_called_once()
+        verify_outputs_wrapped.assert_called_once()
+        verify_sigops_output_wrapped.assert_called_once()
+
+    def test_token_creation_transaction_verify_without_storage(self) -> None:
+        tx = self._get_valid_token_creation_tx()
+
+        verify_pow_wrapped = Mock(wraps=tx.verify_pow)
+        verify_number_of_inputs_wrapped = Mock(wraps=tx.verify_number_of_inputs)
+        verify_outputs_wrapped = Mock(wraps=tx.verify_outputs)
+        verify_sigops_output_wrapped = Mock(wraps=tx.verify_sigops_output)
+
+        with (
+            patch.object(TokenCreationTransaction, 'verify_pow', verify_pow_wrapped),
+            patch.object(TokenCreationTransaction, 'verify_number_of_inputs', verify_number_of_inputs_wrapped),
+            patch.object(TokenCreationTransaction, 'verify_outputs', verify_outputs_wrapped),
+            patch.object(TokenCreationTransaction, 'verify_sigops_output', verify_sigops_output_wrapped),
+        ):
+            tx.verify_without_storage()
+
+        # Transaction methods
+        verify_pow_wrapped.assert_called_once()
+        verify_number_of_inputs_wrapped.assert_called_once()
+        verify_outputs_wrapped.assert_called_once()
+        verify_sigops_output_wrapped.assert_called_once()
+
+    def test_token_creation_transaction_verify(self) -> None:
+        tx = self._get_valid_token_creation_tx()
+
+        verify_pow_wrapped = Mock(wraps=tx.verify_pow)
+        verify_number_of_inputs_wrapped = Mock(wraps=tx.verify_number_of_inputs)
+        verify_outputs_wrapped = Mock(wraps=tx.verify_outputs)
+        verify_sigops_output_wrapped = Mock(wraps=tx.verify_sigops_output)
+        verify_sigops_input_wrapped = Mock(wraps=tx.verify_sigops_input)
+        verify_inputs_wrapped = Mock(wraps=tx.verify_inputs)
+        verify_parents_wrapped = Mock(wraps=tx.verify_parents)
+        verify_sum_wrapped = Mock(wraps=tx.verify_sum)
+        verify_reward_locked_wrapped = Mock(wraps=tx.verify_reward_locked)
+
+        verify_token_info_wrapped = Mock(wraps=tx.verify_token_info)
+
+        with (
+            patch.object(TokenCreationTransaction, 'verify_pow', verify_pow_wrapped),
+            patch.object(TokenCreationTransaction, 'verify_number_of_inputs', verify_number_of_inputs_wrapped),
+            patch.object(TokenCreationTransaction, 'verify_outputs', verify_outputs_wrapped),
+            patch.object(TokenCreationTransaction, 'verify_sigops_output', verify_sigops_output_wrapped),
+            patch.object(TokenCreationTransaction, 'verify_sigops_input', verify_sigops_input_wrapped),
+            patch.object(TokenCreationTransaction, 'verify_inputs', verify_inputs_wrapped),
```

Comment:
It's missing a wrapped call to the `verify_script()` which is usually called by `verify_inputs()`.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/824/commits/50c3f38c8818638bcb201e0956e9e9988ceb7630

Reply:
Thanks!